### PR TITLE
getBootstrapAddresses

### PR DESCRIPTION
**Pull request by peterbraden**

### CODE REVIEW

File: src/bootstrap.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  let bootstrapServerMap = new Map<string, PeerInfo>()
+  let addr: Multiaddr
+
+  if (addrs){
+    addresses = addrs.split(',')
+  } else if (process.env.HOPR_BOOTSTRAP_SERVERS !== undefined) {
+    addresses = process.env.HOPR_BOOTSTRAP_SERVERS.split(',')
+  }  else {
+    // Fall back to DNS
+    let records = await dns.promises.resolveTxt('_dnsaddr.bootstrap.develop.hoprnet.org')
```
Comment (jjperezaguinaga) [inline src/bootstrap.ts:24]: So ideally we have a way to read `ENVIRONMENT` and if not default to `develop`. Right now we have `_dnsaddr.bootstrap.develop` but we also have `_dnsaddr.bootstrap.testnet`, so we need a way to signal when we want which.

File: src/bootstrap.spec.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import assert from 'assert'
+import { getBootstrapAddresses } from './bootstrap'
+
+describe('getBootstrapAddresses', function () {
+  it('passed addresses resolved first', async function () {
+    assert(await getBootstrapAddresses('/ip4/34.65.75.45/tcp/9091/p2p/16Uiu2HAm2cjqsDMmprtN2QKaq3LJrq3YK7vtdbQQFsxGrhLRoYsy'))
+  })
+  it('falls back to dns', async function () {
+    assert(await getBootstrapAddresses())
```
Comment (jjperezaguinaga) [inline src/bootstrap.spec.ts:9]: Not sure what we are testing here, just that the function doesn't throw an error?

Would it be worth observe/spy `dns.promises.resolveTxt` to ensure it was called?

Comment (peterbraden) [reply src/bootstrap.spec.ts:9]: yeah, it's kind of a dumb test, I could definitely make it test better.

File: src/bootstrap.ts (hunk 1 above)
Comment (peterbraden) [reply src/bootstrap.ts:24]: what would you default to?  _dnsaddr.bootstrap.testnet? We can also add an ENV check in addition to manually adding bootstrap servers in the ENV var.

File: src/bootstrap.spec.ts (hunk 2 above)
Comment (peterbraden) [reply src/bootstrap.spec.ts:9]: Will leave as is for now, as we don't use jest in this project so I'd have to add a mocking library.

**Pull request merged**